### DISCUSSION TITLE
What is that tilde ~ character doing?

### DISCUSSION THREAD

**Discussion by metbril** [frenck/home-assistant-config#101] [Q&A] [answered]
In several places in your config you use the tile `~` character. For example:

https://github.com/frenck/home-assistant-config/blob/18e6479d12ab7be6ba508c45be307062894b3ae7/integrations/default_config.yaml#L13

What is that character for? What does it do?

**Comment by miguelarios** [comment]


**Reply by frenck** [reply]
Not in YAML

**Reply by miguelarios** [reply]
Cool

**Comment by frenck** [comment, marked as answer]
@metbril It is an null in YAML

**Comment by tjorim** [comment]
Haha, good try but that has nothing to do with directories. It's a yaml style Frenck preferred to mark null values, the HA docs itself mention this now: https://developers.home-assistant.io/docs/documenting/yaml-style-guide
